gravitee-apim-console-webui/src/management/api/creation/v4/api-creation-v4-confirmation.component.html

```diff
@@ -21,7 +21,7 @@
       <div class="api-creation-confirmation__content__header__icon">
         <mat-icon svgIcon="gio:check"></mat-icon>
       </div>
-      <h2 *ngIf="api">{{ api.name }} has been created</h2>
+      <h2 *ngIf="api">{{ api.name }} has been {{ api.lifecycleState === 'CREATED' ? 'created' : 'deployed' }}</h2>
```

Comment:
I was thinking maybe we should reverse the condition here in order to have `created` by default
```
{{ api.lifecycleState === 'PUBLISHED' ? 'deployed' : 'created' }}
```

Reply:
Yes, good idea!